src/pages/komodo-defi-framework/api/legacy/active_swaps/index.mdx

```diff
@@ -4,10 +4,12 @@ export const description =
 
 # active\_swaps
 
-**active\_swaps (include\_status)**
-
 The `active_swaps` method returns all the swaps that are currently running on the Komodo DeFi Framework API node.
 
+<Note>
+  This method has been deprecated in favor of (active\_swaps v2)\[/komodo-defi-framework/api/v20/active\_swaps]. The legacy version does not include v2 (Trading Protocol Upgrade) swap statuses in responses (it only returns uuids for these).
```

Comment:
Your linking brackets are being escaped here. So on the front-end the link isn't working.

Reply:
The rest of the updates seem to be all in order.